.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+
+jobs:
+  run_tests:
+    name: tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+        python-version: ['3.5','3.8']
+        toxenv: ["django22", "django30", "docs", "quality"]
+        exclude:
+          - python-version: "3.5"
+            toxenv: "django30"
+          - python-version: "3.5"
+            toxenv: "docs"
+          - python-version: "3.5"
+            toxenv: "quality"
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
If we find out that switching to v1 fixes the codecov problems encountered elsewhere, we may want to change this also for consistency.